numpy/distutils/exec_command.py

```diff
@@ -253,7 +253,13 @@ def _exec_command(command, use_shell=None, use_tee = None, **env):
     except EnvironmentError:
         # Return 127, as os.spawn*() and /bin/sh do
         return 127, ''
-    text, err = proc.communicate()
+    try:
+        text, err = proc.communicate()
+    except UnicodeDecodeError:
+        text = ''
+        for byte_line in iter(proc.stdout.readline, ''):
```

Comment:
Is `iter` needed here? For that matter, any reason not to just decode the output at once?